src/coins.ts

```diff
@@ -1914,20 +1010,20 @@ export const findDefaultCoin = (coinKey: CoinKey): Coin => {
 }
 export const findDefaultCoinOnChain = (
   coinKey: CoinKey,
-  chainKey: ChainKey
+  chainId: ChainId
 ): Token => {
```

Comment:
We could rename this function to `findDefaultToken` because it returns a `Token` and not a coin.